[2025-03-13T20:30:17.388+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-13T20:30:17.424+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_pipeline.silver_layer.crm_prd_info scheduled__2025-03-13T20:15:00+00:00 [queued]>
[2025-03-13T20:30:17.442+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_pipeline.silver_layer.crm_prd_info scheduled__2025-03-13T20:15:00+00:00 [queued]>
[2025-03-13T20:30:17.443+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-03-13T20:30:17.465+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): silver_layer.crm_prd_info> on 2025-03-13 20:15:00+00:00
[2025-03-13T20:30:17.476+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=994) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T20:30:17.478+0000] {standard_task_runner.py:63} INFO - Started process 1009 to run task
[2025-03-13T20:30:17.478+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sales_pipeline', 'silver_layer.crm_prd_info', 'scheduled__2025-03-13T20:15:00+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2u0_sgvz']
[2025-03-13T20:30:17.481+0000] {standard_task_runner.py:91} INFO - Job 205: Subtask silver_layer.crm_prd_info
[2025-03-13T20:30:17.550+0000] {task_command.py:426} INFO - Running <TaskInstance: sales_pipeline.silver_layer.crm_prd_info scheduled__2025-03-13T20:15:00+00:00 [running]> on host 2b35ac7f54da
[2025-03-13T20:30:17.717+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='sales_pipeline' AIRFLOW_CTX_TASK_ID='silver_layer.crm_prd_info' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T20:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T20:15:00+00:00'
[2025-03-13T20:30:17.718+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-13T20:30:17.741+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-13T20:30:17.742+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --full-refresh --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/sales --models crm_prd_info']
[2025-03-13T20:30:17.765+0000] {subprocess.py:86} INFO - Output:
[2025-03-13T20:30:22.218+0000] {subprocess.py:93} INFO - [0m20:30:22  Running with dbt=1.9.3
[2025-03-13T20:30:23.348+0000] {subprocess.py:93} INFO - /home/***/.local/lib/python3.12/site-packages/snowflake/connector/options.py:108: UserWarning: You have an incompatible version of 'pyarrow' installed (19.0.1), please install a version that adheres to: 'pyarrow<19.0.0; extra == "pandas"'
[2025-03-13T20:30:23.349+0000] {subprocess.py:93} INFO -   warn_incompatible_dep(
[2025-03-13T20:30:24.480+0000] {subprocess.py:93} INFO - [0m20:30:24  Registered adapter: snowflake=1.9.2
[2025-03-13T20:30:25.852+0000] {subprocess.py:93} INFO - [0m20:30:25  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-13T20:30:25.853+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-13T20:30:25.853+0000] {subprocess.py:93} INFO - - models.sales.Gold_layer
[2025-03-13T20:30:26.343+0000] {subprocess.py:93} INFO - [0m20:30:26  Found 6 models, 35 data tests, 6 sources, 855 macros
[2025-03-13T20:30:26.348+0000] {subprocess.py:93} INFO - [0m20:30:26
[2025-03-13T20:30:26.349+0000] {subprocess.py:93} INFO - [0m20:30:26  Concurrency: 2 threads (target='dev')
[2025-03-13T20:30:26.350+0000] {subprocess.py:93} INFO - [0m20:30:26
[2025-03-13T20:30:27.349+0000] {subprocess.py:93} INFO - [0m20:30:27  1 of 1 START sql incremental model layer_silver.crm_prd_info ................... [RUN]
[2025-03-13T20:31:31.480+0000] {subprocess.py:93} INFO - [0m20:31:31  1 of 1 OK created sql incremental model layer_silver.crm_prd_info .............. [[32mSUCCESS 1[0m in 64.13s]
[2025-03-13T20:31:31.650+0000] {subprocess.py:93} INFO - [0m20:31:31
[2025-03-13T20:31:31.650+0000] {subprocess.py:93} INFO - [0m20:31:31  Finished running 1 incremental model in 0 hours 1 minutes and 5.30 seconds (65.30s).
[2025-03-13T20:31:31.724+0000] {subprocess.py:93} INFO - [0m20:31:31
[2025-03-13T20:31:31.725+0000] {subprocess.py:93} INFO - [0m20:31:31  [32mCompleted successfully[0m
[2025-03-13T20:31:31.725+0000] {subprocess.py:93} INFO - [0m20:31:31
[2025-03-13T20:31:31.726+0000] {subprocess.py:93} INFO - [0m20:31:31  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-13T20:31:33.112+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-13T20:31:33.113+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-13T20:31:33.152+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sales_pipeline, task_id=silver_layer.crm_prd_info, execution_date=20250313T201500, start_date=20250313T203017, end_date=20250313T203133
[2025-03-13T20:31:33.209+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-13T20:31:33.250+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-13T20:31:33.252+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
